tools/swift-refactor/swift-refactor.cpp

```diff
@@ -449,7 +449,7 @@ int main(int argc, char *argv[]) {
     return 1;
   }
 
-  SmallVector<std::unique_ptr<SourceEditConsumer>, 32> Consumers;
```

Comment:
The default here would be quite a bit smaller, if it matters.

Reply:
I believe the only place it's added to is below. So technically this could be 2 if we really cared, but given this is just a testing tool and not even *remotely* hot 🤷